app/models/transaction.rb

```diff
@@ -13,10 +13,14 @@
 
 class Transaction < ActiveRecord::Base
   CURRENCIES = %w(USD RUB)
-  TRANSACTION_TYPES = %w(Residue)
+  TRANSACTION_TYPES = %w(Residue Receipt Transfer)
```

Comment:
@Gataniel you don't need to add additional transaction types. You already have category to check transaction type.